UseCases/Sources/InstrumentsAppTracing/OSSignpostTracing.swift

```diff
@@ -0,0 +1,267 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Tracing open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Swift Tracing project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Baggage
+import Instrumentation
+import Foundation // string conversion for os_log seems to live here
+
+#if os(macOS) || os(tvOS) || os(iOS) || os(watchOS)
+import os.log
+import os.signpost
+
+// ==== ----------------------------------------------------------------------------------------------------------------
+// MARK: OSSignpost Tracing
+
+@available(OSX 10.14, *)
```

Comment:
since signposts are only available since then.
(I may have gotten the availability wrong on other OSes)